### DISCUSSION TITLE
is FancyZones working for you?

### DISCUSSION THREAD

**Discussion by space-artcoder** [jendrikseipp/rednotebook#724] [Q&A]
Since v2.27 I cannot get FancyZones (https://learn.microsoft.com/en-gb/windows/powertoys/fancyzones) to work by pressing shift to show the zones. Am I missing any configuration to get it to work?

**Comment by flinx42** [comment]
Yep, same for me on v2.31 - never seen that happen, other than when apps are running in admin mode.

So I downgraded to v2.26 and FancyZone works again AND I have the internal preview! Might stick to this version until the future catches up with the past 😉
